lib/src/working_copy.rs

```diff
@@ -697,6 +697,19 @@ impl TreeState {
         trace_span!("write tree").in_scope(|| {
             self.tree_id = tree_builder.write_tree();
         });
+        #[cfg(debug_assertions)]
+        {
+            let tree = self
```

Comment:
nit: I slightly prefer `if cfg!(..) {` here since the code block is valid for all profiles.

Reply:
Good point. Fixed.